src/components/MoviesSearchBar/MoviesSearchBar.js

```diff
@@ -10,8 +27,16 @@ const MoviesSearchBar = () => {
           type="text"
           placeholder="What do you want to watch?"
           className="search-movie__bar"
+          onChange={(event) => {
+            setQuery(event.target.value);
+          }}
+          onKeyDown={handleKeypress}
```

Comment:
It is usually better to not name functions like handleKeypress, submitHandler because it is not clear what these functions will do (yes they will handleKeypress but under this can happen everything so another developer will always have to go to exact line where this function had been implemented and read the code). My suggestion is to always name them as what they are doing: e.g. submitHandler -> searchMovies, handleKeypress -> searchMoviesOnEnterPress (BTW it can be combined with searchMovies)